pkg/storj/nodeurl.go

```diff
@@ -116,8 +116,8 @@ func ParseNodeURLs(s string) (NodeURLs, error) {
 // String converts NodeURLs to a string
 func (urls NodeURLs) String() string {
 	var xs []string
-	for _, url := range urls {
-		xs = append(xs, url.String())
+	for _, nodeURL := range urls {
```

Comment:
nit: because it's one line in the for loop instruction I would use just `u` as a var name.